charts/k8ssandra/templates/cassandra/cassdc.yaml

```diff
@@ -116,6 +116,9 @@ spec:
         - "-Dcassandra.system_distributed_replication_dc_names={{ $datacenter.name }}"
         - "-Dcassandra.system_distributed_replication_per_dc={{ min 5 $datacenter.size }}"
 {{- end }}
+{{- end }}
+{{- if not .Values.cassandra.loggingSidecar.enabled }}
+  disableSystemLoggerSidecar: true
```

Comment:
Do we need the inverse here or will the full resource get overwritten anytime there's a modification to our values?